Sources/Controllers/RouteInfoMenu/OARouteInfoView.h

```diff
@@ -18,6 +18,7 @@
 @property (weak, nonatomic) IBOutlet UIButton *goButton;
 @property (weak, nonatomic) IBOutlet UIView *contentContainer;
 @property (weak, nonatomic) IBOutlet UIView *statusBarBackgroundView;
+@property (nonatomic) BOOL selectFromMapTouch;
```

Comment:
Not clear why this field added. It's name is confusing. Looks like it should be dropped.